pkg/render/fluentd.go

```diff
@@ -1040,10 +1042,40 @@ func (c *fluentdComponent) eksLogForwarderDeployment() *appsv1.Deployment {
 		{Name: "AWS_REGION", Value: c.cfg.EKSConfig.AwsRegion},
 		{Name: "AWS_ACCESS_KEY_ID", ValueFrom: secret.GetEnvVarSource(EksLogForwarderSecret, EksLogForwarderAwsId, false)},
 		{Name: "AWS_SECRET_ACCESS_KEY", ValueFrom: secret.GetEnvVarSource(EksLogForwarderSecret, EksLogForwarderAwsKey, false)},
+		relasticsearch.ElasticIndexSuffixEnvVar(c.cfg.ESClusterConfig.ClusterName()),
+		relasticsearch.ElasticUserEnvVar(ElasticsearchEksLogForwarderUserSecret),
+		relasticsearch.ElasticPasswordEnvVar(ElasticsearchEksLogForwarderUserSecret),
+		relasticsearch.ElasticSchemeEnvVar(esScheme),
+		relasticsearch.ElasticHostEnvVar(esHost),
+		relasticsearch.ElasticPortEnvVar(esPort),
+		relasticsearch.ElasticCAEnvVar(c.cfg.OSType),
 	}
 
 	var eksLogForwarderReplicas int32 = 1
 
+	initContainer := corev1.Container{
+		Name:            eksLogForwarderName + "-startup",
+		Image:           c.image,
+		ImagePullPolicy: ImagePullPolicy(),
+		Command:         []string{c.path("/bin/eks-log-forwarder-startup")},
+		Env:             envVars,
+		SecurityContext: c.securityContext(false),
+		VolumeMounts:    c.eksLogForwarderVolumeMounts(),
+	}
+
+	initContainer.Env = append(initContainer.Env, envVars...)
```

Comment:
I don't understand this - this is appending `envVars` onto `envVars`, is it not?